docs/reference/technology/connectivity/cellular.md

```diff
@@ -0,0 +1,157 @@
+## Cellular
+
+Mbed OS cellular provides your IoT application with access to world-wide operator-maintained cellular networks, both IP and non-IP, as Figure 1 illustrates. Mbed OS cellular implementation is based on international 3GPP and OMA standards, and it has been verified to work with all cellular networks such as NB-IoT, CAT-M1, 4G LTE, 3G WCDMA and GPRS.
+
+<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/api-cellular-overview.png)<span>Figure 1. Mbed OS cellular overview</span></span>
+
+Please read about [Mbed OS connectivity technology](https://www.mbed.com/en/technologies/connectivity/) if you are not yet familiar with it.
+
+### Key features
+
+Mbed OS cellular is the preferred choice of building applications for [Mbed Cloud](https://cloud.mbed.com/docs/) and cellular IoT solutions. Key features of Mbed OS cellular include:
+
+- Compatibility with 3GPP TS 27.007 and 27.005.
+- OMA Lightweight M2M Client enabler.
+- Out-of-the-box cellular modem support.
+
+3GPP TS 27.007 and 27.005 are standards to provide consistent cellular network data connectivity and SMS over an AT command interface for cellular devices, regardless of the underlying cellular network.
+
+OMA Lightweight M2M is a protocol from the Open Mobile Alliance for IoT device management. The Mbed OS cellular API provides core functionality to implement the LWM2M Client. For more information, please see _OMA LightweightM2M_ and _OMA LWM2M Object Connectivity Management_ at [OMA Specifications](http://openmobilealliance.org/wp/index.html).
+
+Mbed OS already supports several Mbed Enabled boards with on-board cellular modules out of the box. Because Mbed OS is an open source platform, developers can enable support for new cellular boards with our adaptation framework. Please see our [cellular porting guide](/docs/development/reference/contributing-connectivity.html#cellularinterface) for more information.
+
+### Quick start
+
+There are two phases to Mbed OS connectivity, in general:
+
+1. Connect to a network.
+1. Open a socket to send or receive data.
+
+With cellular, the easiest way to connect your application to the internet over a cellular network is to use the `CellularConnectionFSM` class. It encapsulates most of the complexity of connecting to the cellular network and also reports any changes in connection status to your application. When connected to a cellular network, you can use Mbed OS network sockets as usual, as Figure 2 illustrates.
+
+<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/api-cellular-quick-start.png)<span>Figure 2. Connect to cellular network and open a socket</span></span>
+
+If you want to see code, you can go to our [cellular example](https://github.com/ARMmbed/mbed-os-example-cellular).
+
```

Comment:
We should probably use this one:
https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-cellular/